plugins/services/src/js/structs/ServiceTree.js

```diff
@@ -119,7 +119,10 @@ module.exports = class ServiceTree extends Tree {
    */
   findServiceByName(name) {
     return this.findItem(function(item) {
-      return item.getName() === name && item instanceof Service;
+      return (
+        (item instanceof Framework && item.getFrameworkName() === name) ||
```

Comment:
The only time this could happen is if someone creates an App and adds their own custom label `DCOS_PACKAGE_FRAMEWORK_NAME`. `id` and `DCOS_PACKAGE_FRAMEWORK_NAME` will at most differ by a leading slash and that is the whole reason we need `getFrameworkName`. Cosmos derives the marathon app `id` from `name` at install so if you install with `name: "foo/bar"` or `name: "/foo/bar"`  you'll get `id: "/foo/bar"` in both cases.